tensorboard/manager_test.py

```diff
@@ -23,10 +23,11 @@
 import json
 import os
 import re
+import shutil
 import tempfile
 
 import six
-import tensorflow as tf
+import unittest
```

Comment:
Please revert changes to this file, as described above.

Reply:
Reverting test changes - I'll put those to the side for another PR.